### PR TITLE
[7.59.x] [BXMSPROD-1673] spring-webmvc/spring-framework 5.2.20.RELEASE and spring-boot 2.5.12

### PR DESCRIPTION
**Pull request by rgdoliveira**
https://issues.redhat.com/browse/BXMSPROD-1673

[link](https://www.example.com)

**referenced Pull Requests**
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1936
* https://github.com/kiegroup/optaplanner/pull/1915
*https://github.com/kiegroup/droolsjbpm-integration/pull/2758

You can check Kiegroup organization repositories CI status from [Chain Status webpage](https://kiegroup.github.io/droolsjbpm-build-bootstrap/)

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest</b> (using <i>this</i> e.g. <b>Jenkins retest this</b> optional but no longer required)
 
* for a <b>full downstream build</b> 
  * for <b>jenkins</b> job: please add comment: <b>Jenkins run fdb</b>
  * for <b>github actions</b> job: add the label `run_fdb`
    
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -262,8 +262,8 @@
     <version.org.sonatype.plexus.plexus-sec-dispatcher>1.3</version.org.sonatype.plexus.plexus-sec-dispatcher>
     <version.org.sonatype.sisu>2.3.0</version.org.sonatype.sisu>
     <version.org.sonatype.sisu.sisu-guice>3.2.3</version.org.sonatype.sisu.sisu-guice>
-    <version.org.springframework>5.2.19.RELEASE</version.org.springframework>
-    <version.org.springframework.boot>2.3.10.RELEASE</version.org.springframework.boot>
+    <version.org.springframework>5.3.18</version.org.springframework>
```
Comment (mareknovotny) [inline pom.xml:265]: could we use 5.2.20 instead?

Comment (rgdoliveira) [reply pom.xml]: done

**Review by mareknovotny — APPROVED** [review]

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by Ginxo** [comment]
Jenkins run fdb

**Comment by Ginxo** [comment]
moved to https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1939

**Pull request closed without merging**